src/components/views/spaces/SpacePublicShare.tsx

```diff
@@ -62,6 +63,7 @@ const SpacePublicShare: React.FC<IProps> = ({ space, onFinished }) => {
                         if (onFinished) onFinished();
                         showRoomInviteDialog(space.roomId);
                     }}
+                    aria-label={_t("Invite people")}
```

Comment:
Why can't you use the text in the button itself to select this for tests?

Reply:
If it is fine to write like `cy.findByRole("button", { name: /Invite people/ }).should("exist")`, it is surely possible.

Otherwise you would have to write this way: `cy.findByRole("button", { name: "Invite people Invite with email or username" }).should("exist")` as it looks like strings inside both `<h3>` and `<span>` are treated as the button's text.

I am not sure which one would be better from a viewpoint of actual a11y tool users. Any guidance would be appreciated.